2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,7 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    return cities.map((value, index) => `The temperature in ${value} is ${temperatureService(value)} degrees`);
 }
```

Comment:
Nice use of map! Can you also try converting this into an arrow function?

Reply:
How about this:
`const getTemperatureReport = (cities) => cities.map((value, index) => `The temperature in ${value} is ${temperatureService(value)} degrees`);`

Reply:
Looks good 👍